lib/kafka-worker.rb

```diff
@@ -128,32 +128,37 @@ def on_error(message, err)
   end
 end
 
-if defined?(Rollbar)
-  Rollbar.configure do |config|
-    if ENV['ROLLBAR_ACCESS_TOKEN'] && ['staging', 'production'].include?(ENV['ENV_DOMAIN_NAME'] || Rails.env)
-      config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
-      config.enabled      = true
-    else
-      config.enabled      = false
-    end
+env = ENV['RACK'] || ENV['ENV_DOMAIN_NAME'] || ENV['RACK_ENV'] || ENV['RAILS_ENV'] || 'development'
```

Comment:
seems only be used for raven, so move it down? (generally, I think it's better to avoid those global and very generic names variables)